### PR TITLE
[sync #654] Fixes 654 click2vox frame elements hidden if frame = false

### PR DESCRIPTION
**Pull request by agonza1**
Frame elements hidden if frame = false

### CODE REVIEW

File: views/widget/portal-widget.pug [hunk 1]
```diff
@@ -115,12 +115,12 @@ block content
                             span(style="margin-left:20px" ng-show='widget.show_frame') Pick frame color #[div.glyphicon.glyphicon-tint(tabindex="0" aria-hidden='true' colorpicker colorpicker-position="right" colorpicker-with-input="true" type="text" ng-model="widget.frame_color" ng-blur="generateWidgetCode()" style="cursor: pointer" ng-style="{'color': widget.frame_color || 'black'}")]
 
                       .form-group(analytics-on analytics-event="Extra Configuration" analytics-label="Test user setup")
-                        input(id='test_setup' type='checkbox' name='test_setup' ng-model='widget.test_setup')
+                        input(id='test_setup' type='checkbox' name='test_setup' ng-model='widget.test_setup' ng-disabled='!widget.show_frame')
```
Comment (emilianop11) [inline views/widget/portal-widget.pug]: Isnt the checked/unchecked condition missing here?

File: views/widget/portal-widget.pug [hunk 2]
```diff
@@ -115,12 +115,12 @@ block content
...
                         label(for='test_setup') #[span] Test user setup
                           .badge(data-toggle='tooltip' data-placement="top" title="Place a link to test the users's WebRTC setup") ?
 
                       .form-group(analytics-on analytics-event="Extra Configuration" analytics-label="Show branding")
-                        input(id='show_branding' type='checkbox' name='show_branding' ng-model='widget.show_branding')
+                        input(id='show_branding' type='checkbox' name='show_branding' ng-model='widget.show_branding' ng-disabled='!widget.show_frame')
```
Comment (emilianop11) [inline views/widget/portal-widget.pug]: And here too?

File: public/javascripts/controllers/widget.mixin.js [hunk 3]
```diff
@@ -57,6 +57,14 @@ define([
         e.preventDefault();
       });
 
+      $("input[name = show_frame]").click(function () {
```
Comment (emilianop11) [inline public/javascripts/controllers/widget.mixin.js]: Do we need this? This is also present in click2vox as well. I think we should remove it

Comment (san983) [inline public/javascripts/controllers/widget.mixin.js]: I think we can remove it too

Comment (agonza1) [inline public/javascripts/controllers/widget.mixin.js]: I am doing this to default false to hide test_setup and show_branding elements if show frame is false. If I don't do that we will be showing a widget demo that is not like what the div would generate

Comment (emilianop11) [inline public/javascripts/controllers/widget.mixin.js]: Makes sense

**Pull request merged**